Makefile

```diff
@@ -245,13 +246,15 @@ create-ocp-manifests:
 jenkins-deploy-for-subsystem: ci-deploy-for-subsystem
 
 ci-deploy-for-subsystem: $(VERIFY_CLUSTER) generate-keys
-	export TEST_FLAGS=--subsystem-test && export AUTH_TYPE="rhsso" && export DUMMY_IGNITION=${DUMMY_IGNITION} && WITH_AMS_SUBSCRIPTIONS="True" && \
```

Comment:
@carbonin Am I right to assume you meant `export WITH_AMS_SUBSCRIPTIONS` here?

Reply:
I didn't add this. It was added by @ybettan in b02d18ae8d96187699c578f4ea798dd25b1b1d38

Reply:
Either way, yes I assume this is missing an export given the way it's done elsewhere.